### PR TITLE
Rename "redis" -> "valkey" in transforms.md

### PR DESCRIPTION
**Pull request by rukai**

### CODE REVIEW

File: docs/src/transforms.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ Future transforms won't be added to the public API while in alpha. But in these
...
 | [QueryCounter](#querycounter)                            | ❌          | Alpha                 |
 | [QueryTypeFilter](#querytypefilter)                      | ❌          | Alpha                 |
-| [RedisCache](#rediscache)                                | ❌          | Alpha                 |
-| [RedisClusterPortsRewrite](#redisclusterportsrewrite)    | ❌          | Beta                  |
-| [RedisSinkCluster](#redissinkcluster)                    | ✅          | Beta                  |
-| [RedisSinkSingle](#redissinksingle)                      | ✅          | Beta                  |
+| [ValkeyCache](#rediscache)                               | ❌          | Alpha                 |
+| [ValkeyClusterPortsRewrite](#redisclusterportsrewrite)   | ❌          | Beta                  |
+| [ValkeySinkCluster](#redissinkcluster)                   | ✅          | Beta                  |
+| [ValkeySinkSingle](#redissinksingle)                     | ✅          | Beta                  |
```
Comment (justinweng-instaclustr) [inline docs/src/transforms.md:45]: I think the links to headings need to be updated as well

File: docs/src/transforms.md [hunk 2]
```diff
@@ -480,45 +480,45 @@ This transform will drop messages that match the specified filter. You can eithe
...
     caching_schema:
       test:
         partition_key: [test]
         range_key: [test]
     chain:
-      # The chain can contain anything but must end in a Redis sink
-      - RedisSinkSingle:
+      # The chain can contain anything but must end in a Valkey sink
+      - ValkeySinkSingle:
           # The IP address and port of the upstream redis node/service.
```
Comment (justinweng-instaclustr) [inline docs/src/transforms.md:496]: There are still some `redis` in the comments like this one.

File: docs/src/transforms.md (hunk 1 above)
Comment (rukai) [reply docs/src/transforms.md]: good catch, seems I only replaced upper case `Redis` not `redis`, thanks!

File: docs/src/transforms.md (hunk 2 above)
Comment (rukai) [reply docs/src/transforms.md]: good catch, fixed

**Review by justinweng-instaclustr — APPROVED** [review]

**Review by ronycsdu — APPROVED** [review]

**Pull request merged**